### ISSUE TITLE
Review Order icons needs alt text

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13438]
**Describe the bug**

- Automated tests reveal missing alt text.
- While navigating with VoiceOver (or any Screen Reader), there's no indication of what these links are for. 

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Start the checkout process.
2. Get to the Review Order page.
3. Open VoiceOver
4. Tab through icons.
5. Hear the error: absence of details about the link's purpose.

**Expected behavior**

- alt text needs to be added. e.g. "Edit Payment details, goes to a different page"

**Screenshots**
![Screen Shot 2021-08-05 at 7 15 07 PM](https://user-images.githubusercontent.com/15113219/128569589-5a031181-b578-484d-9c39-dd667fd0bd03.png)

**Comment by saad-mo** [comment]
Since these are font icons, not images the solution is to hide them for SR using `aria-hidden="true"` and use `aria-label="{{ description }}"` on the parent link element to make them accessible. 

The solution proposed from [Font Awesome ](https://fontawesome.com/v4.7/accessibility/) under **If an icon represents an interactive element**

@developpeurweb can you please validate the solution?

**Comment by developpeurweb** [comment]
@saad-mo 
correct, something like this:
```
<a href="" aria-label="Edit Payment details, opens previous page">
  <div class="cx-icon" aria-hidden="true"> </div>
</a>
```

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.
![Review-PaymentDetails](https://user-images.githubusercontent.com/45409515/145466304-540208c6-165f-404e-9c87-fc3497b3ca24.png)
![Review-ShippingAdd](https://user-images.githubusercontent.com/45409515/145466318-63310e4c-6644-4f6e-af6e-7c709f75baee.png)
![Review-deliveryMode](https://user-images.githubusercontent.com/45409515/145466325-3c6f0ee0-8e0a-4ebe-adef-010bc00dee01.png)
